client/src/components/TACourse.jsx

```diff
@@ -138,53 +138,57 @@ export const TACourse = ({ netId, taCourses, setSelectedCourse }) => {
           <TabPanels>
             <TabPanel>
               <h1 className="title">Start Queue</h1>
-              <p className="description">
-                Choose the course that you are hosting office hour for:
-              </p>
-              <Select
-                mt = '1rem'
-                marginBottom='20%'
-                width='75%'
-                focusBorderColor='#918fe1'
-                background='white'
-                value={selectedValue}
-                placeholder='Choose a course:'
-                onChange={e => setSelectedValue(parseInt(e.target.value))}
-              >
-                {options}
-              </Select>
+              <div style={{ 'maxWidth': '75%' }}>
```

Comment:
i notice a lot of inline css, can consider moving stuff into *.css files?

Reply:
Great find Izzy!